dev/com.ibm.ws.org.apache.cxf.cxf.rt.rs.client.3.2/bnd.overrides

```diff
@@ -13,7 +13,7 @@ Import-Package: \
   !org.springframework.*,\
   !javax.cache.*,\
   !org.osgi.service.blueprint.*,\
-  *
+   *
```

Comment:
No need to add a space here - probably Eclipse did it to "help" you when you were making another change, but now that there are no other changes, it would be better to remove this change from the commit to keep the history cleaner.